src/sass/components/_product-box.scss

```diff
@@ -80,3 +80,11 @@
     align-items: center;
   }
 }
+
+@for $i from 1 through 8 {
+  .col-3:nth-child(#{$i}) {
+    .photo {
+      background-image: url("../images/pictures/ch_#{$i}.jpg");
```

Comment:
where images are? I don't see it in this commit / PR.

Reply:
Sorry, forgot to tick one box while commit, to attach folder with images. All fixed now, did push.